tests/Aspire.Hosting.Tests/Aspire.Hosting.Tests.csproj

```diff
@@ -59,6 +57,7 @@
     <Compile Include="$(RepoRoot)src\Aspire.Hosting.Nats\NatsContainerImageTags.cs" />
     <Compile Include="$(RepoRoot)src\Aspire.Hosting.Milvus\MilvusContainerImageTags.cs" />
     <Compile Include="$(RepoRoot)src\Aspire.Hosting.MongoDB\MongoDBContainerImageTags.cs" />
+    <Compile Include="$(RepoRoot)src\Aspire.Hosting.MySql\MySqlContainerImageTags.cs" />
```

Comment:
What is this still used for? It seems like it should be removed. `Aspire.Hosting.Tests` shouldn't need to be testing all the extension resources we create.

Reply:
`VerifyTestProgramFullManifest` checks a manifest which has all these resources, and uses 

```c#
"image": "{{TestConstants.AspireTestContainerRegistry}}/{{MySqlContainerImageTags.Image}}:{{MySqlContainerImageTags.Tag}}",

```

Reply:
I think this line should be removed.